src/Application/JoomlaApplicationSite.php

```diff
@@ -20,6 +23,8 @@ class JoomlaApplicationSite extends JApplicationCms
 {
     public static function initializeApplication() {
         self::$instances['site'] = new self();
+//        include_once 'JoomlaApplicationAdministrator.php';
+//        self::$instances['administrator'] = new JoomlaApplicationAdministrator();
```

Comment:
Hi @ifandpromiss 

if the commented codes will never be used or useful in the future, just remove them